mmcv/runner/runner.py

```diff
@@ -27,14 +27,16 @@ class Runner(object):
         work_dir (str, optional): The working directory to save checkpoints
             and logs.
         log_level (int): Logging level.
+        logger: Custom logger. If `None`, use default logger.
```

Comment:
We need to make the docstring consistent with the following format.

argument name (argument type): some text

Reply:
Is it `logging.Logger`?

Reply:
Yep.